content/ways-to-train.md

```diff
@@ -42,208 +39,89 @@ keywords:
-        Salary: "Yes"
-        Funding available: "No"
-        Length of training: "1 year"
-        Available for: "Primary, secondary"
-        Qualification: "QTS"
-      - "Postgraduate teaching apprenticeship"
-  undergraduate:
-    name: feature_table
-    arguments:
-      - Cost: "Typically £9,250 per year"
-        Salary: "No"
-        Funding available: "Yes"
-        Length of training: "1 to 4 years"
-        Available for: "Primary, secondary"
-        Qualification: "Qualifications vary — for example, BA (Hons) with QTS, or BSc (Hons) with QTS"
-      - "Undergraduate teacher training"
-  teach-first:
-    name: feature_table
-    arguments:
-      - Cost: "You do not pay tuition fees"
-        Salary: "Yes"
-        Funding available: "No"
-        Length of training: "2 years"
-        Available for: "Primary, secondary"
-        Qualification: "PGDE with QTS"
-      - "Teach First"
-  assessment-only:
-    name: feature_table
-    arguments:
-      - Available for: "Primary, secondary, further education"
-        Qualification: "QTS"
-      - "Assessment only"
----
-
-You need to get 'qualified teacher status' (QTS) to teach in the majority of schools in England. This includes primary, secondary and special ‘maintained schools’ - part of the state funded schools system. 
-
-You can get QTS in different ways.
-
-For example, if you already have an undergraduate degree you can do a postgraduate initial teacher training (ITT) course which leads to QTS.
-
-If you do not have a degree, you can do an undergraduate degree which leads to QTS.
-
-Different types of training have different eligibility criteria, [funding options](/funding-your-training) and qualifications. Lots of courses also allow you to study part-time.
 
-You can also [get support to improve your subject knowledge](/guidance/become-a-teacher-in-england#subject-knowledge-enhancement-courses) to supplement your training.
-
-Not looking for primary or secondary training? Consider training to teach [early years](/early-years-teaching-training) or [further education](/further-education-teacher-training).
-
-$get-a-tta$
-
-$pgce$
-
-PGCE programmes combine academic study with a minimum of 24 weeks spent training in schools.
-
-You do not need an academic qualification such as a PGCE to teach in England, but it may be useful if you want to teach in another country.
-
-Many PGCE courses give you credits that count towards a master’s degree.
+---
 
-You can also [get funding](/funding-your-training).
+## Train to get qualified teacher status
 
-#### Postgraduate Diploma in Education
+You need ‘qualified teacher status’ (QTS) to work in the majority of schools in England including state maintained primary, secondary and special schools.
 
-Some providers offer a Postgraduate Diploma in Education (PGDE) instead of a PGCE. A PGDE sometimes offers more masters credits than a PGCE.
+There are different ways to get qualified teacher status, for example:
```

Comment:
We have treated jump links very differently on other pages - this list demotes the jump links and means more reading required to understand where to go? One to discuss with design? E.g. see funding your training for jump links established style.

Reply:
as I mentioned earlier, I'm very much in need of formatting help for this whole page - have asked Gary for help